src/components/accounts/recovery_setup/SetupRecoveryMethod.js

```diff
@@ -48,14 +49,16 @@ const StyledContainer = styled(Container)`
         margin-top: 40px;
         font-weight: 600;
         font-size: 15px;
+        display: flex;
+        align-items: center;
     }
 
 `
 
 class SetupRecoveryMethod extends Component {
 
     state = {
-        option: 'phrase',
+        option: this.props.router.location.method || 'phrase',
```

Comment:
:nit: I think that as we move into container/presentational components, we're going to want this to be a prop named e.g. `method` rather than leak the awareness of `router` into this component (so we can render it as a storybook without injecting the router into our presentational component)